### PR TITLE
Improvement: module name must not be equal to the package name.

### PR DESCRIPTION
**Pull request by coderimus**
### Description (*)

This PR provides the next changes:

- Think that the plugin should also provide good practices and based on this I suggest not to allow to create a module with the same name as a package :) I added condition which checks whether the module's name equals to the package's name

image

- Add a blank line at the end of the composer.json line

image

- Add tooltips to the  New Module form

image

I choose the 1.0.0-develop as a destination branch because it looks like an improvement which should not break a release) 

### Fixed Issues (if relevant)
N/A

### Questions or comments
N/A

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages

### CODE REVIEW

File: resources/magento2/inspection.properties [hunk 1]
```diff
@@ -1,2 +1,15 @@
 inspection.plugin.duplicateInSameFile=The plugin name already used in this file. For more details see Inspection Description.
-inspection.plugin.duplicateInOtherPlaces=The plugin name "{0}" for targeted "{1}" class is already used in the module "{2}" ({3} scope). For more details see Inspection Description.
\ No newline at end of file
+inspection.plugin.duplicateInOtherPlaces=The plugin name "{0}" for targeted "{1}" class is already used in the module "{2}" ({3} scope). For more details see Inspection Description.
+must.not.be.empty.should.contain.letters.or.numbers=Must not be empty, should contain letters or numbers
+package.name=Package Name
```
Comment (VitaliyBoyko) [inline resources/magento2/inspection.properties:4]: I believe these messages aren't related to any of inspections.

Comment (coderimus) [reply resources/magento2/inspection.properties]: @VitaliyBoyko thank you for your review. I investigated the form edit functionality deeper and think updated PR should be better now)

File: resources/magento2/new_module_generation_dialog.properties [hunk 2]
```diff
@@ -0,0 +1,10 @@
+must.not.be.empty.should.contain.letters.or.numbers=Must not be empty, should contain letters or numbers
```
Comment (VitaliyBoyko) [inline resources/magento2/new_module_generation_dialog.properties:1]: Let's move this phrase to `validator` properties and rest to `common`. I think they can be reused in other forms.

**Comment by coderimus** [comment]
@VitaliyBoyko requested changes implemented.
Should we have a new blank line at the end of the file in the properties?

**Comment by VitaliyBoyko** [comment]
Sure, let's have the empty line

**Comment by coderimus** [comment]
@VitaliyBoyko new lines added.

**Comment by VitaliyBoyko** [comment]
QA passed ✅ 
image

**Pull request merged**